src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+.logo {
+  height: 40px;
+  width: 40px;
+  left: 50px;
+  top: 10px;
+  margin-left: 50px;
+}
+
+.header {
```

Comment:
header styles should go first, not logo

Reply:
also, remove margins for body
<img width="1139" alt="Screenshot 2022-05-03 at 14 12 34" src="https://user-images.githubusercontent.com/57598825/166443314-81e63d40-38de-416d-a576-c3f775aedfce.png">